lib/puppet/util/errors.rb

```diff
@@ -37,11 +37,11 @@ def adderrorcontext(error, other = nil)
   # @return [String] description of file and line
   def error_context
     if file and line
-      " at #{file}:#{line}"
+      _(" at %{file}:%{line}") % { file: file, line: line }
```

Comment:
I know, I hate these little "build up the error message" blocks. I guess we can see what the translator makes of it? Or leave it out entirely...